pkg/kubectl/cmd/util/helpers.go

```diff
@@ -402,6 +402,7 @@ func AddDryRunFlag(cmd *cobra.Command) {
 func AddServerSideApplyFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool("server-side", false, "If true, apply runs in the server instead of the client. This is an alpha feature and flag.")
 	cmd.Flags().Bool("force-conflicts", false, "If true, server-side apply will force the changes against conflicts. This is an alpha feature and flag.")
+	cmd.Flags().String("applier", "kubectl", "Name of the applier used to track field ownership. This is an alpha feature and flag.")
```

Comment:
For alpha flags we add `experimental-` as a prefix.

Reply:
The three flags here are experimental, should I rename them all?